pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -104,20 +102,21 @@ def statistics_reset(context):
 @click.pass_obj
 def statistics_server_on(context):
     """
-    Enable statistics on current server.
+    Enable server maintained statistics.
 
-    This command  activates the gathering of statistics in the WBEM
-    server and the return of server response times for inclusion in
-    pywbemcli maintained statistics. The gathering of server statistics and the
-    returning of server response times may not be implemented by all WBEM
-    servers.
+    This command activates the gathering of statistics in the current
+    WBEM server and also activates the returning of server response times in
```

Comment:
Probably don't need the work "also"

Reply:
NOT CHANGED. See other occurrence of this.

Reply:
We agreed in the call today this is ok to go.